webapp/src/components/conference/conference.tsx

```diff
@@ -54,6 +54,20 @@ export default class Conference extends React.PureComponent<Props, State> {
         return Math.max(document.documentElement.clientHeight || 0, window?.innerHeight || 0) - (BORDER_SIZE * 2);
     }
 
+    preventMessages = (event: MessageEvent) => {
+        if (!this.props.post || !this.api) {
+            return;
+        }
+        const meetingURL = new URL(this.props.post.props.meeting_link);
+        if (event.origin !== meetingURL.origin && event.data) {
+            const data = JSON.parse(event.data || '{}');
+            if (data.postis === true && data.scope.indexOf('jitsi_meet_external_api_') === 0) {
```

Comment:
The postis handler just checks for truthy; we should match that here to avoid bypasses.
```suggestion
            if (data.postis && data.scope.indexOf('jitsi_meet_external_api_') === 0) {
```